### PR TITLE
Huawei Push

### PR DESCRIPTION
**Pull request by davidbielik**
TODO:

- [x] Update screenshots
- [x] Add code samples
- [x] ...
- [ ] Profit

Note: Feature to be released on 6/30 with Major Dashboard Release

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/huawei_integration.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+## 3. Enter Huawei Credentials in the Braze Dashboard
+
+Using the credentials from your Huawei Developer account, enter the `Huawei App ID` and `Huawei App Secret` within your newly created Huawei Braze App. When you're done, save your changes.
+
+![Huawei Dashboard Credentials][12]
+
+## 4. Integrate the Braze Android SDK into your App
+
+_hocus pocus julian's focus_
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/huawei_integration.md:41]: Will get to this next week

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/huawei_integration.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+---
+nav_title: Huawei Integration
+platform: Android
+page_order: 1
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/huawei_integration.md:4]: This shouldn't be the first page. Maybe last

**Comment by radixdev** [comment]
@davidbielik added the android instructions

**Comment by davidbielik** [comment]
hey @KellieHawks this is ready for review whenever you're ready (no rush)

**Pull request merged**